ui/app/helpers/utils/conversions.util.js

```diff
@@ -171,14 +171,23 @@ export function addHexes (aHexWEI, bHexWEI) {
   })
 }
 
```

Comment:
The changes in this file don't change the functionality of `sumHexWEIsToRenderableFiat`, while making part of the logic previously in `sumHexWEIsToRenderableFiat` exportable via `sumHexWEIsToUnformattedFiat`